test/manual/definitions/apm/ruby/rubyonrails.json

```diff
@@ -0,0 +1,27 @@
+{
+    "global_tags": {
+      "owning_team": "virtuoso",
+      "Environment": "development",
+      "Department": "product",
+      "Product": "virtuoso"
+    },
+    "resources": [
+      {
+        "id": "ruby-on-rails-linux2",
+        "provider": "aws",
+        "type": "ec2",
+        "size": "t2.medium",
+        "ami_name": "al2023-ami-2023.3.20240122.0-kernel-6.1-x86_64",
+        "user_name": "ec2-user"
+      }
+    ],
+    "services": [
+      {
+        "id": "rubyrails",
+        "local_source_path": "/mnt/deployer/",
```

Comment:
I think here key has to be source path and value is open-install-library git repo. what do you think?